apps/authentication/models.py

```diff
@@ -19,7 +19,7 @@ class Users(db.Model, UserMixin, AuditMixin):
     __tablename__ = 'users'
     id = db.Column(db.Integer, primary_key=True)
     username = db.Column(db.String(64), unique=True)
-    uid = db.Column(db.String(15), unique=True)
+    uid = db.Column(db.String(15), unique=True, default=generate_uid)
```

Comment:
Already being done on line 48
```suggestion
    uid = db.Column(db.String(15), unique=True)
```